pkg/config/data.go

```diff
@@ -121,8 +121,17 @@ type File struct {
 }
 
 // Read the File and populate its fields from the filename on disk.
-func (f *File) Read(filename string) error {
-	_, err := toml.DecodeFile(filename, f)
+func (f *File) Read(fpath string) error {
+	// G304 (CWE-22): Potential file inclusion via variable.
+	// gosec flagged this:
+	// Disabling as we need to load the config.toml from the user's file system.
+	// This file is decoded into a predefined struct, any unrecognised fields are dropped.
+	/* #nosec */
+	bs, err := os.ReadFile(fpath)
```

Comment:
Should this be `ioutil.ReadFile()` to be consistent with all the other changes int he PR?